crates/swc_ecma_utils/src/lib.rs

```diff
@@ -3063,6 +3063,10 @@ where
     }
 
     fn visit_mut_var_declarator(&mut self, n: &mut VarDeclarator) {
+        if !n.name.is_ident() {
```

Comment:
We should remove the whole `visit_mut_var_declarator` function.
We should only skip the ident declartion.